oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/MinLongMetrics.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.analysis.metrics;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.skywalking.oap.server.core.analysis.metrics.annotation.Entrance;
+import org.apache.skywalking.oap.server.core.analysis.metrics.annotation.MetricsFunction;
+import org.apache.skywalking.oap.server.core.analysis.metrics.annotation.SourceFrom;
+import org.apache.skywalking.oap.server.core.storage.annotation.Column;
+
+/**
+ * @author jian.tan
+ */
+
+@MetricsFunction(functionName = "min")
+public abstract class MinLongMetrics extends Metrics implements LongValueHolder {
+
+    protected static final String VALUE = "value";
+
+    @Getter @Setter @Column(columnName = VALUE, isValue = true) private long value;
+
+    @Entrance
+    public final void combine(@SourceFrom long count) {
+        if (this.value == 0) {
+            this.value = count;
+        }
```

Comment:
An obvious bug here, try the test codes:

```java
        MinLongMetricsImpl impl = new MinLongMetricsImpl();
        impl.combine(10);
        impl.combine(0);
        impl.combine(10000);
```

I expect `impl.getValue()` to be `0`, but it's `10000`, please recheck @JaredTan95

Reply:
Making `value` default to `Long.MAX_VALUE` is my first intuition

Reply:
> An obvious bug here, try the test codes:
> 
> ```java
>         MinLongMetricsImpl impl = new MinLongMetricsImpl();
>         impl.combine(10);
>         impl.combine(0);
>         impl.combine(10000);
> ```
> 
> I expect `impl.getValue()` to be `0`, but it's `10000`, please recheck @JaredTan95

I suggest putting this into unit test as well

Reply:
Thanks to your experienced guidance, I will update it.